components/story-inquiry-page/StoryInquiryPage.js

```diff
@@ -0,0 +1,190 @@
+import Navigation from '../navigation/Navigation';
+import Footer from '../footer/Footer';
+import H1Styled from '../h1-styled/H1Styled';
+import { getString } from '../../lib/richText';
+import StoryConceptFaq from './story-concept-faq/StoryConceptFaq';
+import PageDescription from '../page-description/PageDescription';
+import {
+  SquiggleWavy,
+  H1StyledContainer,
+  LongLinesDesktop,
+  LongLinesFlippedDesktop,
+  LongLinesTablet,
+  LongLinesFlippedTablet,
+  LongLinesContainer,
+  TopSparkArrowContainer,
+  BottomSparkArrowContainer,
+  SideImageContainer,
+  SideImage,
+  QuoteCarouselContainer,
+} from './StoryInquiryPage.styles';
+import StoryInquiryForm from './story-inquiry-form/StoryInquiryForm';
+import { icons } from '../../style/icons';
+import LongSparkArrow from '../long-spark-arrow/LongSparkArrow';
+import QuoteCarousel from '../quote-carousel/QuoteCarousel';
+
+export default function StoryInquiryPage({
```

Comment:
If it's an easy fix, let's make the text portions more narrow like the hifi shown below
![Uploading Screen Shot 2021-05-05 at 11.42.31 AM.png…]()